node/config/def.go

```diff
@@ -102,6 +102,14 @@ func DefaultFullNode() *FullNode {
 	}
 }
 
+func DefaultStorageWorker() *StorageWorker {
+	cfg := &StorageWorker{}
+	cfg.API.Protocol = "http"
+	cfg.API.ExtractRoutableIP = true
+	cfg.API.RemoteListenAddress = "0.0.0.0:3456"
```

Comment:
Maybe use multiaddrs here ?

E.g. `"/ip4/0.0.0.0/tcp/0"`

Reply:
this is not a place for multiaddr. I would not like to change the current behaviour of the system